### DISCUSSION TITLE
Welcome to griffon Discussions!

### DISCUSSION THREAD

**Discussion by aalmiray** [griffon/griffon#314] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swalton00** [comment]
How do we create a new Griffon application now that lazybones is gone (no longer available from bintray)?

**Reply by aalmiray** [reply]
There are 3 options at the moment:
- use the Maven archetype if the target build tool is Maven.
- copy from an existing project and modify. 
- start from scratch. 

Unfortunately there is no suitable tool that provides the same behavior as Lazybones. There's a fork of the project called Skeletor but I think is not yet released.

**Comment by swalton00** [comment]
I'm using a MutableDataSourceConfiguration to set the required properties. Those values are saved in a properties file which is read at startup. If the properties are empty, I open a window to set the application preferences. The problem is the main  window opens and completely covers the preferences window. I tried set the preferences window to application modal but that that doesn't work if I set it in the view initialization (probably because of the timing of the main window opening. I can't seem to hide the main window either. Do you have any suggestions as to how to force the preferences window into the foreground (and set it modal if possible).
